client/start.js

```diff
@@ -169,7 +169,10 @@ export function startApp(renderApplication, reducers, opts) {
     }
 
     store.dispatch({ type: NAVIGATE_TO_PAGE, page, currentPath: path });
-
+    if( opts.enableFCM ) {
+      const mssgSenderId = get(page, ["config", "fcmMessageSenderId"], null);
```

Comment:
typo

Reply:
```
if opts.enableFCM &&  get(page, ["config", "fcmMessageSenderId"], null)
```

Also, please update the readme with FCM instructions

Reply:
Done